handbook/engineering/README.md

```diff
@@ -876,9 +876,9 @@ The following rituals are engaged in by the directly responsible individual (DRI
 | Release testing/QA | Every three weeks | Every release cycle, by end of day Wednesday of release week, all issues move to "Ready for release" on the 🚀Release board. | Reed Haynes |
 | Release testing/QA report | Every three weeks | Every release cycle, on the Monday of release week, the DRI for the release ritual is updated on status of testing. | Reed Haynes |
 
-## Fleet Sandbox Oncall
+## 24/7 on-call
 
-Oncall for Sandbox will happen in shifts of 1 week. The people involved in them will be:
+The 24/7 on-call (aka infrastructure on-call) team is responsible for alarms related to fleetdm.com, Fleet sandbox, Fleet managed cloud, as well as delivering 24/7 support for Fleet Ultimate customers.  The infrastructure (24/7) on-call responsibility  happen in shifts of 1 week. The people involved in them will be:
```

Comment:
@zayhanlon clarified this on-call responsibility includes first line responses for 24/7 customer support, and added sentence casing.  Also switched to present tense ("is" instead of "will be")- though I missed some spots.  (I realize some of these were overdue changes from prior to this PR, just FYI)

Reply:
@mikermcneil  I don't think it was clear (to me at least) to the team that this is also for customer support for Fleet, the product itself.

Reply:
Gotcha, thanks for clarifying.  Yes, this is first response for infra as well as anything outside of business hours.